### PR TITLE
Update usage to reflect default of Mesh CA for new installs

### PR DESCRIPTION
**Pull request by djazayeri**

### CODE REVIEW

File: scripts/asm-installer/install_asm
Comment (zerobfd) [inline scripts/asm-installer/install_asm:172]: Can we change "MODE=install" to "--mode install"? Caps vs not is optional, but in the CA field we use lowercase so we should be standard. We shouldn't use an equal sign since the script doesn't support using it.

File: scripts/asm-installer/install_asm
Comment (zerobfd) [inline scripts/asm-installer/install_asm:176]: If it's required for migrate mode, let's not put the (optional) tag in front.

**Review by zerobfd — APPROVED** [review]
LGTM after a couple of small style points

File: scripts/asm-installer/install_asm
Comment (djazayeri) [reply scripts/asm-installer/install_asm:172]: Sure. I originally tried this but it led to an unfortunate line break which made things a bit confusing. But I'll do it that way, since it is in fact correct.

File: scripts/asm-installer/install_asm
Comment (djazayeri) [reply scripts/asm-installer/install_asm:176]: I am following the precedent from

 -k|--key_file          <FILE PATH>  (optional) The key file for a service
                                      account. Required if --service_account is
                                      passed.

**Comment by elfinhe** [comment]
do we know why the test is failing? I can either help make an exception merge if we have confidence that it is a flaky run.

File: scripts/asm-installer/install_asm
Comment (zerobfd) [reply scripts/asm-installer/install_asm:176]: That's a good point. When I wrote it, service account and key file are both optional, so it made sense. This one seems different to me because the dependency is on a mandatory option, not an optional one.

If this might be confusing for users, I'd honestly rather just remove the (optional) from the beginning altogether and let users read the descriptions for more info. I don't see a way to have the two cases be consistent to everyone otherwise.

I won't block on this if you think this is the right way to handle it.

**Comment by zerobfd** [comment]
We don't know why the test is failing--I'm looking at it now but don't have an idea on how long it will take. Since it's just a usage change I think an exception merge should be fine--runs before this PR also failed and I'm pretty doubtful this is the culprit.

**Pull request merged**
